pkg/env/postgres_query_tracer.go

```diff
@@ -12,6 +12,9 @@ var (
 	// PostgresQueryTracerQueryThreshold sets a threshold for how long an individual Postgres query must take to be logged
 	PostgresQueryTracerQueryThreshold = registerDurationSetting("ROX_POSTGRES_QUERY_TRACER_QUERY_THRESHOLD", 200*time.Millisecond)
 
-	// PostgresQueryLogger toggles whether to trace Postgres queries and their timing
+	// PostgresQueryLogger toggles whether to log Postgres queries for debugging
 	PostgresQueryLogger = RegisterBooleanSetting("ROX_POSTGRES_QUERY_LOGGER", false)
+
+	// PostgresKeepTestDB does not destroy the test databases for debugging
+	PostgresKeepTestDB = RegisterBooleanSetting("ROX_POSTGRES_TEST_KEEP_DB", false)
```

Comment:
Will it appear in the release? Maybe we should have it set in tag quarded code and in `release` set it to false

Reply:
Let's leave it for now.  I find it unlikely but we could have a desire to use this if for some reason a UT of the rel build code is failing.